### PR TITLE
Update README.spanish.md: Contenido nuevo y fixes

### PR DESCRIPTION
**Pull request by 13luismb**
En el presente commit se añade:
-Lista de contribuidores
-Traducción inicial de la sección 8, referente a Docker
-Estandarización de mensajes de "Volver arriba"
-Enlace de menús del archivo

### CODE REVIEW

**Comment by goldbergyoni** [comment]
@13luismb Welcome aboard and thanks for the amazing PR! 🚀

Sorry for the long review time, we're now on it

@kevynb Shall we ask from one of the Spanish contributors to review? Feel confident to review ourselves?

**Comment by kevynb** [comment]
Well I think @desko27 will be more qualified than me to review Spanish changes. If he's ok with it, I'll merge.

Thanks @13luismb for taking the time to PR those changes!

File: README.spanish.md [hunk 1]
```diff
@@ -1039,6 +1037,42 @@ Ten en cuenta que con la introducción del nuevo motor V8 junto con los nuevos e
 
 <br/><br/><br/>
 
+<p align="right"><a href="#table-of-contents">⬆ Volver arriba</a></p>
+
+# `8. Mejores prácticas de Docker`
+
+🏅 Muchas gracias a [Bret Fisher](https://github.com/BretFisher) de quién hemos aprendido muchas de las siguientes prácticas
```
Comment (desko27) [inline README.spanish.md:1044]: See https://llevatilde.es/palabra/qui%C3%A9n

```suggestion
🏅 Muchas gracias a [Bret Fisher](https://github.com/BretFisher) de quien hemos aprendido muchas de las siguientes prácticas
```

**Review by desko27 — APPROVED** [review]
Just a little comment, but LGTM in general. Thank you for contributing @13luismb!

**Review by kevynb — APPROVED** [review]

**Comment by kevynb** [comment]
@all-contributors please add @13luismb for translation

**Comment by kevynb** [comment]
Let's merge this! 🎉🎉

**Pull request merged**